README.md

```diff
@@ -31,30 +32,30 @@
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/Harshal662"><img src="https://avatars.githubusercontent.com/u/79760384?v=4?s=100" width="100px;" alt="Harshal Abak"/><br /><sub><b>Harshal Abak</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=Harshal662" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/thesilentline"><img src="https://avatars.githubusercontent.com/u/82605925?v=4?s=100" width="100px;" alt="Divyanshi Singh Bora"/><br /><sub><b>Divyanshi Singh Bora</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=thesilentline" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/Jean-BaptisteC"><img src="https://avatars.githubusercontent.com/u/87148630?v=4?s=100" width="100px;" alt="Jean-Baptiste"/><br /><sub><b>Jean-Baptiste</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=Jean-BaptisteC" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/akj2018"><img src="https://avatars.githubusercontent.com/u/43956935?v=4?s=100" width="100px;" alt="Akshay Jain"/><br /><sub><b>Akshay Jain</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=akj2018" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/jack-white9"><img src="https://avatars.githubusercontent.com/u/83393304?v=4?s=100" width="100px;" alt="Jack White"/><br /><sub><b>Jack White</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=jack-white9" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/viktoriussuwandi"><img src="https://avatars.githubusercontent.com/u/68414300?v=4?s=100" width="100px;" alt="Viktorius Suwandi"/><br /><sub><b>Viktorius Suwandi</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=viktoriussuwandi" title="Code">💻</a></td>
-    </tr>
-    <tr>
-      <td align="center" valign="top" width="14.28%"><a href="https://xzm27-d5f48.web.app"><img src="https://avatars.githubusercontent.com/u/55028818?v=4?s=100" width="100px;" alt="XZM27"/><br /><sub><b>XZM27</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=Xzm27" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/SaketKaswa20"><img src="https://avatars.githubusercontent.com/u/105808363?v=4?s=100" width="100px;" alt="Saket Kaswa"/><br /><sub><b>Saket Kaswa</b></sub></a><br /><a href="https://github.com/tungbq/devops-basic/commits?author=SaketKaswa20" title="Code">💻</a></td>
-    </tr>
-  </tbody>
-</table>
-
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
-
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->

```

Comment:
@viktoriussuwandi it seems the content of contributors section were changed too. Could you please check and only change the `gertting-started` part and keep other part as is?

Reply:
yes sure